src/utils/string.ts

```diff
@@ -0,0 +1,3 @@
+export const getNormalizedString = (str: string, toLowerCase: boolean) => {
```

Comment:
`getNormalizedNode` as before would be better.

Reply:
The issue is that the sort comparison works on strings that are derived from the node (per the `getNameForSort` function). Mainly that's to support sorting on local names, which act as a tiebreaker in the case where the same import has multiple local names. For example:

```
import {a, a as b, a as c, d} from './z';
```

the names used for sorting (and also for the error message) are `a`, `a as b`, `a as c`, and `d`, and they're in the correct alphabetical order.